dv/uvm/core_ibex/Makefile

```diff
@@ -41,6 +41,9 @@ SIGNATURE_ADDR      := 8ffffffc
 ### Ibex top level parameters ###
 IBEX_CONFIG         := opentitan
 
+# Path to DUT used for coverage reports
```

Comment:
Is this value something you would want to override on the command line? If not, it might just be cleaner to hardcode it in `metadata.py`.

Reply:
Likely not but I'd prefer not to hardcode it into the script as it's specific to the testbench structure where the scripts are a more general flow that could be used with multiple testbench setups. So in absence of a better place to put such things I'd like to leave it here.